resources/views/match/createMatch.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layouts.plain')
+@section('title','Create Match')
+
+@section('content')
+    <div class="container" id="app">
```

Comment:
add a place to show form verification errors - make it an included file so we can show it in different places